package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@datadog/native-appsec",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
Can we leave out the version bump out of this PR ?

Reply:
pushed